functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py

```diff
@@ -541,3 +542,70 @@ def test012_check_cloned_vm(self):
 
         with self.assertRaises(HTTPError) as e:
              self.api.cloudapi.machines.rollbackSnapshot(machineId=cloned_vm_id, epoch=snapshotEpoch)
+
+    def test013_check_IP_conflict(self):
+        """ OVC-043
+        *Test case for checking machine connectivity through external network*
+
+        **Test Scenario:**
+
+        #. Create CS1, should succeed
+        #. Create VM1,VM2 to CS1,should succeed.
```

Comment:
#. Create cloud space CS1, should succeed
#. Create two virtual machines (VM1 & VM2) in CS1, should succeed.